packages/components/src/components/contract-card/contract-card-items/accumulator-card-body.tsx

```diff
@@ -1,13 +1,35 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isCryptocurrency, getLimitOrderAmount, isValidToSell } from '@deriv/shared';
+import { TContractInfo } from '@deriv/shared/src/utils/contract/contract-types';
 import ContractCardItem from './contract-card-item';
 import ToggleCardDialog from './toggle-card-dialog';
 import Icon from '../../icon';
 import MobileWrapper from '../../mobile-wrapper';
 import Money from '../../money';
 import { ResultStatusIcon } from '../result-overlay/result-overlay';
+import { ProposalOpenContract, ContractUpdate } from '@deriv/api-types';
```

Comment:
In packages/shared/src/utils/contract/contract-types.ts `TContractInfo` 'extends' `ProposalOpenContract`:
```
export type TContractInfo = ProposalOpenContract & {
    contract_update?: ContractUpdate;
};
```
Do we need to import `ProposalOpenContract` if `TContractInfo` includes the same?
Just asking

Reply:
I guess not. will include the change.